_posts/randy/2022-06-24-reflecting-on-the-future.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title:  "Reflecting on the Future - A Mirror Sample of Thinks to Come"
+permalink: /@randy/reflecting-on-the-future
+ext_image: "assets/img/sample-in-progress.png"
+description: I am Old School human; equipped with analog gears but upgraded with digital belts. Having a fair amount of past, I spend a lot of time reflecting on the past. It seems like I always have a need to measure and gauge my present with the paths and events it took to get me here.
+tags: tails analog-sampling
+---
+
+![](/assets/img/sample-in-progress.png)
+
+<!-- <video width="720" height="480" controls>
```

Comment:
As mentioned in message, I think MPEG-4/H.264 or webm should fix this issue

Reply:
https://user-images.githubusercontent.com/66967414/175633284-a2964a2f-9be7-42a2-af5d-a6ff1a6cd11d.mp4

Reply:
This is h.264 replacement file - will need renaming and relocation